bug-bounty-list/bug-bounty-list.json

```diff
@@ -1701,8 +1701,8 @@
   },
   {
     "program_name": "Cobalt",
-    "policy_url": "https://cobalt.io/cobalt",
-    "submission_url": "",
+    "policy_url": "https://cobalt.io/security/practices",
+    "submission_url": "security@cobalt.io",
     "launch_date": "",
     "cash_reward": true,
```

Comment:
Should this still be `true`? I didn't see a direct mention of cash rewards on the newly linked page.

Reply:
I didn't see anything that made me feel confident in changing it either way. Down to do what you think is best.

Reply:
The page mentions an email address and

> Since launching Cobalt, we’ve invited anyone on the internet to notify us of issues they might find in our application to further strengthen and secure our platform. All vulnerability report submissions are read within hours of receipt, and we aim to respond to all submissions within 48 hours.

I'd say go with `false` for cash rewards unless there's an explicit mention of cash compensation for vulnerability reports.

Reply:
_updated_